docs/cloud-native-security/get-started-with-kspm.asciidoc

```diff
@@ -236,3 +236,136 @@ The *Add agent* wizard helps you deploy the KSPM integration on the Kubernetes c
 2. Apply the manifest using the `kubectl apply -f` command. For example: `kubectl apply -f elastic-agent-managed-kubernetes.yaml`
 
 After a few minutes, a message confirming the {agent} enrollment appears, followed by a message confirming that data is incoming. You can then click *View assets* to see where the newly-collected configuration information appears throughout {kib}, including the <<findings-page,Findings page>> and the <<cloud-posture-dashboard, Cloud Posture dashboard>>.
+
+[discrete]
+[[kspm-eck]]
+=== Set up KSPM on ECK deployments
+When running https://www.elastic.co/guide/en/cloud-on-k8s/current/k8s-quickstart.html[ECK] deployment, KSPM benchmark will is not support out-of-the-box.
+A patch is required to https://www.elastic.co/guide/en/cloud-on-k8s/current/k8s-elastic-agent-configuration.html[Elastic Agent CRD] and https://www.elastic.co/guide/en/cloud-on-k8s/current/k8s-elastic-agent-configuration.html#k8s-elastic-agent-role-based-access-control[Elastic Agent Cluster-Role]
+
+.Patch Elastic Agent
+[%collapsible]
+====
+Add `volumes` and `volumeMonunts` to `podTemplate`
+```yaml
+podTemplate:
+  spec:
+    containers:
+    - name: agent
+      volumeMounts:
+      - name: proc
+        mountPath: /hostfs/proc
+        readOnly: true
+      - name: cgroup
+        mountPath: /hostfs/sys/fs/cgroup
+        readOnly: true
+      - name: varlibdockercontainers
+        mountPath: /var/lib/docker/containers
+        readOnly: true
+      - name: varlog
+        mountPath: /var/log
+        readOnly: true
+      - name: etc-full
+        mountPath: /hostfs/etc
+        readOnly: true
+      - name: var-lib
+        mountPath: /hostfs/var/lib
+        readOnly: true
+      - name: etc-mid
+        mountPath: /etc/machine-id
+        readOnly: true
+    volumes:
+```
+====
+
+.Patch RBAC
+[%collapsible]
```

Comment:
I liked how in the previous example, you wrote "Add `volumes` and `volumeMounts` to `podTemplate`:", would it make sense to add a similar short description of the change in the RBAC .yaml as well?